### PR TITLE
chore(ci): add connector node docker

### PR DESCRIPTION
**Pull request by huangjw806**
I hereby agree to the terms of the [RisingWave Labs, Inc. Contributor License Agreement](https://gist.github.com/TennyZhuang/f00be7f16996ea48effb049aa7be4d66#file-rw_cla).

## What's changed and what's your intention?

add connector node in docker image and release assets.
add connector node to playground

## Checklist

- [x] I have written necessary rustdoc comments
- [x] I have added necessary unit tests and integration tests
- [x] I have added fuzzing tests or opened an issue to track them. (Optional, recommended for new SQL features).
- [x] I have demonstrated that backward compatibility is not broken by breaking changes and created issues to track deprecated features to be removed in the future. (Please refer to the issue)
- [x] All checks passed in `./risedev check` (or alias, `./risedev c`)

### CODE REVIEW

File: ci/scripts/release.sh [hunk 1]
```diff
@@ -54,6 +56,12 @@ if [[ -n "${BUILDKITE_TAG+x}" ]]; then
   echo "--- Release upload asset"
   tar -czvf risingwave-"${BUILDKITE_TAG}"-x86_64-unknown-linux.tar.gz risingwave
   gh release upload "${BUILDKITE_TAG}" risingwave-"${BUILDKITE_TAG}"-x86_64-unknown-linux.tar.gz
+
+  echo "--- Release build and upload risingwave connector node jar asset"
+  git clone https://"$GITHUB_TOKEN"@github.com/risingwavelabs/risingwave-connector-node.git
+  cd risingwave-connector-node && git checkout ${CONNECTOR_NODE} && mvn -B package -Dmaven.test.skip=true
+  cd assembly/target && mv risingwave-connector-1.0.0.tar.gz risingwave-connector-"${BUILDKITE_TAG}".tar.gz
```
Comment (fuyufjh) [inline ci/scripts/release.sh:63]: The version is fixed at `1.0.0` ? 🤣

File: ci/scripts/docker.sh [hunk 2]
```diff
@@ -3,13 +3,16 @@
 # Exits as soon as any line fails.
 set -euo pipefail
 
-# Build docker image ${BUILDKITE_COMMIT}-${arch}
-
-date="$(date +%Y%m%d)"
 ghcraddr="ghcr.io/risingwavelabs/risingwave"
 dockerhubaddr="risingwavelabs/risingwave"
 arch="$(uname -m)"
+CONNECTOR_NODE=$(cat ./connector-node)
```
Comment (fuyufjh) [inline ci/scripts/docker.sh:9]: How about renaming it to `connector-node-version` to make it more clear?

**Review by fuyufjh — APPROVED** [review]
LGTM for the rest.

File: ci/scripts/release.sh (hunk 1 above)
Comment (huangjw806) [reply ci/scripts/release.sh:63]: 😂

**Review by tabVersion — APPROVED** [review]
lets roll for the first release

File: ci/scripts/release.sh (hunk 1 above)
Comment (StrikeW) [reply ci/scripts/release.sh:63]: Actually we can add an arg `-Drelease.version=0.1.16` to `mvn -B package -Dmaven.test.skip=true` to specify the output version of the tar file.

File: src/risedevtool/src/task/connector_service.rs [hunk 3]
```diff
@@ -33,7 +33,7 @@ impl ConnectorNodeService {
     fn connector_path(&self) -> Result<PathBuf> {
         let prefix_bin = env::var("PREFIX_BIN")?;
         Ok(Path::new(&prefix_bin)
-            .join("connector-node")
+            .join("../../../../ci/connector-node")
```
Comment (StrikeW) [inline src/risedevtool/src/task/connector_service.rs:36]: Why you need to modify this line? It needs to be the `$PREFIX_BIN/connector-node` directory.

**Review by StrikeW — APPROVED** [review]
connector node added to playground.

**Pull request merged**